src/Bundle.js

```diff
@@ -28,6 +28,7 @@ export default class Bundle {
 		});
 
 		this.entry = unixizePath( options.entry );
+		this.entryId = null;
```

Comment:
I tried to, but `this.entryModule` is `null` at this point. This is because it is still being fetched (and it won't be completed until all its dependencies are fetched too).